python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -237,13 +249,9 @@ def run(self):
                     )
                 if not continue_sched or self._shutdown:
                     break
-            for callback in get_execution_callbacks(self._data_context):
-                callback.after_execution_succeeds()
```

Comment:
Just to confirm my understanding -- when the main thread receives the interrupt signal, the streaming executor starts shutting down and breaks out of the while loop, and that's why it calls `after_execution_succeeds`?

Reply:
that's right